gym/wrappers/monitoring/video_recorder.py

```diff
@@ -6,42 +9,52 @@
 import subprocess
 import tempfile
 from io import StringIO
+from typing import Optional, Union
 
 import numpy as np
 
 from gym import error, logger
 
 
-def touch(path):
+def touch(path: str):
+    """Touch a filename at path."""
     open(path, "a").close()
 
 
 class VideoRecorder:
-    """VideoRecorder renders a nice movie of a rollout, frame by frame. It
-    comes with an `enabled` option so you can still use the same code
-    on episodes where you don't want to record video.
+    """VideoRecorder renders a nice movie of a rollout, frame by frame.
+
+    It comes with an `enabled` option, so you can still use the same code on episodes where you don't want to record video.
```

Comment:
Maybe use double backticks for enabled? 
```suggestion
    It comes with an ``enabled`` option, so you can still use the same code on episodes where you don't want to record video.
```